raytracing/tests/testsImagingPath.py

```diff
@@ -354,6 +354,14 @@ def testChiefRayInfiniteFOVNoY(self):
         with self.assertRaises(ValueError):
             path.chiefRay()
 
+    def testBugNegativeFNumber(self):
+        path = ImagingPath()
+        path.append(System4f(f1=40, diameter1=25.4, f2=150, diameter2=50, label="Excitation relay"))
```

Comment:
quicly, this was a system that made a negative f/#. Single lenses didn't seem to work and 4f system neither.